.github/ISSUE_TEMPLATE/mMaintainerRequest.md

```diff
@@ -34,8 +41,13 @@ PLEASE REMOVE ALL COMMENTS ONCE YOU HAVE READ THEM.
 
 ## Existing Package Details
 
+<!-- You may leave the URLs empty as long as the issue title matches the identifier of the package -->
+
 Package URL:
 Package source URL:
+
 <!-- please remove the following section if you are the current maintainer of the package, otherwise fill out the details -->
+<!-- Please also ensure that the date the maintainer was contacted is using the ISO 8601 standard (YYYY-MM-DD) -->
+
 Date the maintainer was contacted:
```

Comment:
Should we / can we add the date format to this? ie: Date the maintainer was contacted (in YYYY-MM-DD format):? 

It's in the instructions but we know how well people read those ....

Reply:
Sure, just let me check what I need to change to support that first then.